src/planwise/client/current_project/subs.cljs

```diff
@@ -60,12 +59,12 @@
 (register-sub
  :current-project/transport-time
  (fn [db [_]]
-   (reaction (get-in @db [:current-project :transport :time]))))
+   (reaction (get-in @db [:current-project :project-data :filters :transport :time]))))
 
 (register-sub
  :current-project/demand-map-key
  (fn [db [_]]
-   (reaction (get-in @db [:current-project :demand-map-key]))))
+   (reaction (get-in @db [:current-project :project-data :demand-map-key]))))
```

Comment:
Though not being used currently, `demand-map-key` is not actually an attribute of the project per se, but of its region+filters. Should we still keep it in `project-data`?